draft-ietf-quic-http.md

```diff
@@ -1527,8 +1527,10 @@ Where HTTP/2 employs PADDING frames and Padding fields in other frames to make a
 connection more resistant to traffic analysis, HTTP/3 can either rely on
 transport-layer padding or employ the reserved frame and stream types discussed
 in {{frame-reserved}} and {{stream-grease}}.  These methods of padding produce
-different results in terms of the granularity of padding, the effect of packet
-loss and recovery, and how an implementation might control padding.
+different results in terms of the granularity of padding, how padding is
+arranged in relation to the information that is being protected, whether
+padding is applied in the case of packet loss, and how an implementation might
+control padding.
```

Comment:
How about a citation to https://datatracker.ietf.org/doc/draft-wood-pearg-website-fingerprinting/

Reply:
That seems like a different problem.

Reply:
PR sent.